doc/source/graph/svcorch.md

```diff
@@ -166,4 +158,18 @@ Note no metrics or extra data will be added to the request so this would need to
 
 ## Routing in Metadata
 
-The current default orchestrator in Go the "executor" does not return routing meta data in request calls. This is a [known issue](https://github.com/SeldonIO/seldon-core/issues/1823). 
\ No newline at end of file
+The current default orchestrator in Go the "executor" does not return routing meta data in request calls. This is a [known issue](https://github.com/SeldonIO/seldon-core/issues/1823). 
+
+
+## Java Engine [DEPRECATED]
+
+.. Attention:: 
+   The Java version of the service orchestrator has now been deprecated and
+   will be completely removed in future versions of Seldon Core.
+   We strongly suggest **migrating to the new version of the service
+   orchestrator** (implemented in Go) before upgrading.
+   Note that the new version of the service orchestator is enabled by default
+   since Seldon Core `1.1`.
+
+For further details on the Java engine see previous versions of this page in the docs.
```

Comment:
Worth adding direct link to previous release docs?

Reply:
To which one though? I'd leave it up to the user to choose the right version for their docs.

Reply:
I'd say last one that contained it. Though it's not mandatory.